src/seedu/addressbook/AddressBook.java

```diff
@@ -210,6 +210,10 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
+        runProgram();
+    }
+
+    private static void runProgram() {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.